bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VLF_LO/ttH_HTT_Singlet_LO/ttH_HTT_Singlet_LO_run_card.dat

```diff
@@ -256,7 +257,7 @@
 #*********************************************************************
 # Jet measure cuts                                                   *
 #*********************************************************************
- 0   = xqcut   ! minimum kt jet measure between partons
+ 20   = xqcut   ! minimum kt jet measure between partons
```

Comment:
can you remind me the conclusion on the xqcut and qcut in pythia?

Reply:
Hello,
In a discussion with Aurore we decided to start with both xqcut and qcut as 20. We agreed that xqcut=0 was not good, but the value 20 is just a value we thought was reasonable ( I never used pythia before).

Reply:
I think xqcut=10 is appropriate and will give you some flexibility if needed